### DISCUSSION TITLE
which linux wm has same features as yabai?

### DISCUSSION THREAD

**Discussion by molleweide** [asmvik/yabai#1067] [Q&A] [answered]
Hi there guys,

I am currently a macos user and I want to learn how to become more comfortable with linux desktop. Therefore I am asking myself which window manager could do the same things as yabai but on linux. 

Does anyone have anything to recommend here or is this a stupid question? I am now looking at suckless's dwm because it seems
rather lightweight but I haven't really looked into all of its features yet.

**Comment by 24unix** [comment]
i3 is very popular, else just search for tiling window manager linux.

**Reply by molleweide** [reply]
mm yeah, i agree, I see many alternatives by searching. I was more wondering if there are any particular features such as spaces or stacks that are specifically only available on a particular wm on linux?

**Comment by asmvik** [comment, marked as answer]
Bumping an old discussion here, but **yabai** is the macOS equivalent of **bspwm** for linux.
